src/app/clients/clients-routing.module.ts

```diff
@@ -147,9 +149,35 @@ const routes: Routes = [
             }]
           }
         ]
+      },
+      {
+        path: ':clientId',
+        data: { title: extract('Client View'), routeParamBreadcrumb: 'clientId' },
```

Comment:
@BLasan san The path `clientId` has already been mentioned in line number 53, you need to nest the routes with the present defined path. In this way, it will create confusion for the server as will cause failure in the system. Please change it and nest with the declared path.

Reference: The `paycharge` path will be under the `general` path.

Reply:
> @BLasan san The path `clientId` has already been mentioned in line number 53, you need to nest the routes with the present defined path. In this way, it will create confusion for the server as will cause failure in the system. Please change it and nest with the declared path.
> 
> Reference: The `paycharge` path will be under the `general` path.

Nested routings added earlier as you mentioned. But it will produce an output as follow:
![Screenshot from 2020-05-12 17-25-00](https://user-images.githubusercontent.com/43112139/81685374-a34c6a00-9475-11ea-9d34-a94ad0e9a942.png)

@karantakalkar  and I were having a conversion above based on this problem. Couldn't find a better answer than this.

Reply:
If this preview is okay then I can make the nested routings as you mentioned :)

Reply:
Yes, this thing occurred for even other modules I had been working on. We need to discuss with the senior mentors and finalized the design. Will inform you, as soon as it's updated.

Reply:
Okay :+1:

Reply:
> > @BLasan san The path `clientId` has already been mentioned in line number 53, you need to nest the routes with the present defined path. In this way, it will create confusion for the server as will cause failure in the system. Please change it and nest with the declared path.
> > Reference: The `paycharge` path will be under the `general` path.
> 
> Nested routings added earlier as you mentioned. But it will produce an output as follow:
> ![Screenshot from 2020-05-12 17-25-00](https://user-images.githubusercontent.com/43112139/81685374-a34c6a00-9475-11ea-9d34-a94ad0e9a942.png)
> 
> @karantakalkar and I were having a conversion above based on this problem. Couldn't find a better answer than this.

@BLasan You can go ahead with the implementation of this view.

Reply:
Ok. I'll implement as in this screenshot

Reply:
@muskankhedia Please take a look